.gitrepo

```diff
@@ -0,0 +1,12 @@
+; DO NOT EDIT (unless you know what you are doing)
```

Comment:
This seems weird? Why did this end up in the root folder?

Reply:
hmm. i'll revert